packages/openapi3/src/lib.ts

```diff
@@ -117,6 +125,17 @@ const EmitterOptionsSchema: JSONSchemaType<OpenAPI3EmitterOptions> = {
       description:
         "If the generated openapi types should have the `x-typespec-name` extension set with the name of the TypeSpec type that created it.\nThis extension is meant for debugging and should not be depended on.",
     },
+    "safeint-strategy": {
+      type: "string",
+      enum: ["double-int", "int64"],
+      nullable: true,
+      default: "int64",
+      description: [
+        "How to handle safeint type. Options are:",
```

Comment:
same.  Is this auto-generated?

Reply:
This one has to be written manually until we have the config as tsp